.github/workflows/release_pytorch_wheels.yml

```diff
@@ -7,14 +7,25 @@ permissions:
   contents: read
 
 jobs:
-  build_pytorch_grid:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      version_suffix: ${{ steps.set.outputs.version_suffix }}
+    steps:
+      - id: set
+        run: echo "version_suffix=$(date +'%Y%m%d')" >> $GITHUB_OUTPUT
+
+  release_pytorch:
+    needs: setup
     strategy:
+      fail-fast: false
       matrix:
         python_version: ["3.11", "3.12"]
-        families: ["gfx110X"]
+        target: [gfx942]
```

Comment:
This gfx target should come from a workflow input so we can trigger on demand with any target(s)